### PR TITLE
Request to fix broken link

### PR DESCRIPTION
**Pull request by sasaito**
There is an invalid link redirecting to non existing page.

Repro steps:
Go to https://docs.microsoft.com/en-us/microsoftteams/security-compliance-overview
Click the link "Compliance Framework for Industry Standards and Regulations"

Expected results:
Link redirect to an existing page or pdf file

Actual results:
404 - file or directory not found error occurred on https://download.microsoft.com/download/1/4/3/1434ABAB-B8E9-412D-8C3A-187B5FCB7A2F/Compliance%20Framework%20document.pdf

* The following URL shows the pdf file published in March 2017.
https://download.microsoft.com/download/B/2/7/B27B3EF3-8849-4C18-8BA4-5AD755728620/Compliance%20Framework_customer%20guidance.pdf

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/security-compliance-overview.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Microsoft Teams is built on the Office 365 hyper-scale, enterprise-grade cloud,
...
 
-Services in compliance categories C and D that have industry-leading compliance commitments are enabled by default. Services in categories A and B come with controls to turn on or turn off these services for an entire organization. Details can be found in the [Compliance Framework for Industry Standards and Regulations](https://go.microsoft.com/fwlink/?linkid=855777). Teams also supports Cloud Security Alliance compliance.
+Services in compliance categories C and D that have industry-leading compliance commitments are enabled by default. Services in categories A and B come with controls to turn on or turn off these services for an entire organization. Details can be found in the [Compliance Framework for Industry Standards and Regulations](https://go.microsoft.com/fwlink/?linkid=855777 invalid link redirecting to non existing page.). Teams also supports Cloud Security Alliance compliance.
```
Comment (nenonix) [inline Teams/security-compliance-overview.md]:
```suggestion
Services in compliance categories C and D that have industry-leading compliance commitments are enabled by default. Services in categories A and B come with controls to turn on or turn off these services for an entire organization. Details can be found in the [Compliance Framework for Industry Standards and Regulations](https://download.microsoft.com/download/B/2/7/B27B3EF3-8849-4C18-8BA4-5AD755728620/Compliance%20Framework_customer%20guidance.pdf). Teams also supports Cloud Security Alliance compliance.
```

**Comment by scanum** [comment]
@sasaito Please review requested changes.

**Comment by LolaJacobsen** [comment]
@MicrosoftHeidi could you review this one please?

**Comment by nenonix** [comment]
@LolaJacobsen  @MicrosoftHeidi  See https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/pull/2603

**Comment by scanum** [comment]
@LolaJacobsen @MicrosoftHeidi PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**